myguessgame.html

```diff
@@ -1,33 +1,27 @@
-<!-- // How many day are left in the year 2015 (current date 5/30/15) (246 days answer) -->
+
 <script>
 	(function()  {
-		var answer = '246';
-		var guess = prompt ('How many days are left in the current calender year?');
-		// while (answer != 246) {
-		// 	guess =	Number.parseInt(guess); }
-		// while (true) 
-		// 	if ()
-		// 		(guess > 246)
-		// 		alert ('Sorry thats not the right answer. Hint less days')
-		// 		break;
-		// 		else if (guess < 246)
-		// 		prompt('Sorry not quite, guess again')
-
-
-		// 		alert('Correct! way to go!')
-		if (answer == guess) {
-			alert('Wahooo!! your correct!');
-		} else if (answer > guess) {
-			alert('Sorry you guess is less than the answer');
-		} else if (answer < guess){
-			alert('Sorry that\'s too high');
-		} else {
-			alert('random');
-		}
-
+		var answer = 246;
+		var guess = prompt('How many days are left in the current calender year?');
+		var guesscounter = 0;
+		guess = Number.parseInt(guess);
+		while (guesscounter <=3) {
+			if (guess === answer) {
```

Comment:
Keep an eye on your spacing with indents. It looks like your Tabs are not set to Indent Using Spaces (Google "Sublime Indent Using Spaces Default Settings").

It probably looks fine in your Sublime editor, but when you commit you can see how it pushes everything way out.